frontend/__tests__/containers/ReportStep/ReportStep.test.tsx

```diff
@@ -685,4 +687,63 @@ describe('Report Step', () => {
       expect(closeNotification).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('error notification', () => {
+    beforeEach(() => {
+      jest.spyOn(echarts, 'init').mockImplementation(
+        () =>
+          ({
+            setOption: jest.fn(),
+            resize: jest.fn(),
+            dispatchAction: jest.fn(),
+            dispose: jest.fn(),
+            //eslint-disable-next-line @typescript-eslint/no-explicit-any
+          }) as any,
+      );
+    });
+
+    it('should correctly render dora chart', async () => {
```

Comment:
As a reader, from the description of this case, my understanding of this case is, in some scenario the notification should be in the view.
So I suppose under the prerequisites, assertion like `expect(errorAlertElement).toBeVisible()` is better?

Reply:
Aye, in this case the notification should be called only once to indicate the csv file expiration time, since the page is successfully loaded. Thus I added a new expect to ensure the addnotification is called with the expire message